code/modules/uplink/uplink_items/nukeops.dm

```diff
@@ -410,31 +410,61 @@
 // Explosives and Grenades
 // ~~ Grenades ~~
 
-/datum/uplink_item/explosives/grenades
+/datum/uplink_item/explosives/military_grade
 	cost = 15
 	surplus = 35
 	purchasable_from = UPLINK_ALL_SYNDIE_OPS
 
-/datum/uplink_item/explosives/grenades/buzzkill
+/datum/uplink_item/explosives/military_grade/buzzkill
 	name = "Buzzkill Grenade Box"
 	desc = "A box with three grenades that release a swarm of angry bees upon activation. These bees indiscriminately attack friend or foe \
 		with random toxins. Courtesy of the BLF and Tiger Cooperative."
 	item = /obj/item/storage/box/syndie_kit/bee_grenades
 
-/datum/uplink_item/explosives/grenades/virus_grenade
+/datum/uplink_item/explosives/military_grade/virus_grenade
 	name = "Fungal Tuberculosis Grenade Box"
 	desc = "A primed bio-grenade packed into a compact box. Comes with five Bio Virus Antidote Kit (BVAK) \
 		autoinjectors for rapid application on up to two targets each, a syringe, and a bottle containing \
 		the BVAK solution."
 	item = /obj/item/storage/box/syndie_kit/tuberculosisgrenade
 	restricted = TRUE
 
-/datum/uplink_item/explosives/grenades/viscerators
+/datum/uplink_item/explosives/military_grade/viscerators
 	name = "Viscerator Delivery Grenade Box"
 	desc = "A box containing unique grenades that deploys a swarm of viscerators upon activation, which will chase down and shred \
 		any non-operatives in the area."
 	item = /obj/item/storage/box/syndie_kit/manhack_grenades
 
+/datum/uplink_item/explosives/military_grade/frag_grenade
+	name = "Frag Grenade Box"
+	desc = "A box containing 7 frag grenades. Simple. Destructive."
+	item = /obj/item/storage/box/syndie_kit/frag_grenades
+	cost = 5
+
+/datum/uplink_item/explosives/military_grade/x4
+	name = "Composition X-4"
+	desc = "Similar to C4, but with a stronger blast that is directional instead of circular. X-4 can be placed on a solid surface, such as a wall or window, \
+		and it will blast through the wall, injuring anything on the opposite side, while being safer to the user. For when you want a controlled explosion that \
+		leaves a wider, deeper, hole."
+	item = /obj/item/grenade/c4/x4
+	cost = 2
+
+/datum/uplink_item/explosives/military_grade/syndicate_minibomb
+	name = "Syndicate Minibomb"
+	desc = "The minibomb is a grenade with a five-second fuse. Upon detonation, it will create a small hull breach \
+			in addition to dealing high amounts of damage to nearby personnel."
+	item = /obj/item/grenade/syndieminibomb
+	cost = 2
```

Comment:
Not sure if intentional, but you've given clown ops minibombs here (which they don't have access as of now)